Seasons Defense/Assets/Scripts/Tower.cs

```diff
@@ -0,0 +1,29 @@
+using UnityEngine;
+
+public class Tower : MonoBehaviour
+{
+    public GameObject missilePrefab;
+    public float offset;
+    private Vector3 towerPosition;
```

Comment:
For the sake of convention and local sanity. Private instance variables should have the following syntax:
```suggestion
    private Vector3 _towerPosition;
```